pkg/chunk/aws/fixtures.go

```diff
@@ -59,15 +65,20 @@ func dynamoDBFixture(provisionedErr, gangsize, maxParallelism int) testutils.Fix
 	return fixture{
 		name: fmt.Sprintf("DynamoDB chunks provisionedErr=%d, ChunkGangSize=%d, ChunkGetMaxParallelism=%d",
 			provisionedErr, gangsize, maxParallelism),
+		kind: testutils.Table,
 		clients: func() (chunk.IndexClient, chunk.ObjectClient, chunk.TableClient, chunk.SchemaConfig, error) {
 			dynamoDB := newMockDynamoDB(0, provisionedErr)
 			schemaCfg := chunk.SchemaConfig{
 				Configs: []chunk.PeriodConfig{{
 					IndexType: "aws",
-					From:      model.Now(),
+					From:      model.Now().Add(-time.Hour * 2),
```

Comment:
I'm still bothered by lots of expressions like this and `model.Now().Add(-time.Hour))`.  I have to work through all the logic to try to figure out what is the difference and/or similarity.  If that computation is interesting, can we give it a symbolic name? 

Elsewhere we use `github.com/weaveworks/common/mtime` to force "now" to particular values during testing; does that help?

Reply:
The need for this comes from the [code](https://github.com/cortexproject/cortex/blob/4c9e2025ab6733e0f42187b64d2522d558f9f31c/pkg/chunk/testutils/testutils.go#L85) used to generate chunks for tests. I don't think it adds to much complexity to use the `Add` syntax. I will see if there is an easy way to generalize this for all fixtures.